freqtrade/data/metrics.py

```diff
@@ -194,32 +194,35 @@ def calculate_cagr(days_passed: int, starting_balance: float, final_balance: flo
     return (final_balance / starting_balance) ** (1 / (days_passed / 365)) - 1
 
 
-def calculate_expectancy(trades: pd.DataFrame) -> float:
+def calculate_expectancy(trades: pd.DataFrame) -> Tuple[float, float]:
     """
     Calculate expectancy
     :param trades: DataFrame containing trades (requires columns close_date and profit_abs)
-    :return: expectancy
+    :return: expectancy, expectancy_ratio
     """
-    if len(trades) == 0:
-        return 0
 
-    expectancy = 1
+    expectancy = 0
+    expectancy_ratio = float('inf')
```

Comment:
i'm ... sceptical about using `inf` here

it's prone to cause `ValueError: Out of range float values are not JSON compliant` - and is the expectancy really infinite if we did no trade, or is it rather 0?

Reply:
I'm using frog's default value of inf. Personally, I prefer 100.

Reply:
Then let's go with that.
backtesting will be ... picky otherwise :laughing: